slamon_afm/tests/afm_test.py

```diff
@@ -27,3 +27,31 @@ def setUp(self):
     def tearDown(self):
         db.drop_all()
         self.app_context.pop()
+
+
+class ConfigUtilTest(TestCase):
+    def testOkTrueValues(self):
+        for value in ('True', 'TRUE', 'true', 'Yes', 'yes', 'YES'):
+            self.assertTrue(_bool_from_str(value))
+
+    def testOkFalseValues(self):
+        for value in ('False', 'FALSE', 'false', 'No', 'no', 'NO'):
+            self.assertFalse(_bool_from_str(value))
+
+    def testNokValues(self):
+        for value in ('f', 't', '0', '1', 'nope', 'yep'):
+            with self.assertRaises(ValueError):
+                _bool_from_str(value)
+
+    def testOkLogLevels(self):
+        # known values (standard logging levels)
+        for value in ('CRITICAL', 'Error', 'warning', 'INFO', 'DEBUG', 'NOTSET'):
```

Comment:
I think this test would be better if formatted as:

```python
for value in [("CRITICAL", 50), ("ERROR", 40), ("WARNING", 30), ("INFO", 20), ("DEBUG", 10), ("NOTSET", 0)]:
    self.assertEqual(value[1], _log_level_from_str(value[0])
```

Now the test has (almost) exactly the same logic the function has you are trying to test: `getattr(logging, value.upper())` vs you are testing against `getattr(logging, value.upper(), None)`